Rubberduck.VBEEditor/SafeComWrappers/Abstract/IWindow.cs

```diff
@@ -5,6 +5,7 @@ namespace Rubberduck.VBEditor.SafeComWrappers.Abstract
 {
     public interface IWindow : ISafeComWrapper, IEquatable<IWindow>
     {
+        IntPtr RealHwnd { get; set; }
```

Comment:
I thought about that, but there were 2 issues - First, the *actual* HWnd (even when it does exist) isn't ready soon enough, because apparently the VBE creates the window first, and *then* instantiates the underlying object.  The `RealHwnd` finds the handle from the OS instead of relying on the VBE object to be ready to supply it.  Second, it's conceivable that the HWnd of the backing window can change - I don't remember exactly what I was doing at the time (maybe something with docking or maximizing\minimizing in the MDI?), but I watched as the VBE destroyed a window and then recreated it with a fresh hwnd.  Keeping both of them available (at least internally) allows a good check for that contingency.  The `HWnd` *getter* is a different story, but my final rational was that until we can raise something like "OnHwndChange", we might as well allow callers to make the comparison if they need to.  Again, WIP.